### PR TITLE
Lc 16/sign up page markup

### PR DESCRIPTION
**Pull request by Azimkhan93**
Sign up page is done
image
Login page is done
image

### CODE REVIEW

File: .eslintrc.cjs
Comment (AnvarAka1) [inline .eslintrc.cjs]: no-console should be an error

File: src/modules/auth/component/Login/Login.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/Login/Login.tsx]: Remove this comment

File: src/modules/auth/component/Login/Login.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/Login/Login.tsx]: Can we have this Typography as a Component, if it is used in several pages. What do you think?

File: src/modules/auth/component/Login/Login.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/Login/Login.tsx:48]: If sm and lg are the same, just use sm={5}. remove lg

File: src/modules/auth/component/Login/Login.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/Login/Login.tsx]: This button is used in many places. It makes sense to create a custom component named "Button" or "PrimaryButton"

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/SignUp/SignUp.tsx]: Why don't you use a custom component with this style? Name it as TextField and use MuiTextField inside of it. All the styles will be there too

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/SignUp/SignUp.tsx]: Avoid using marginTop on a component itself. Try to wrap it in Grid for example or Box and use spacing/margin.
The presentational component like Button, TextField, Typography and etc should NOT use the margins. You can of course use them sometimes, but it is more like an exception to the rule, not the rule.

Especially for the components, that are custom. They should not have the margins/paddings. If you want to make a component universal/generic, you need to avoid using paddings/margins on them. Instead, you import them, wrap the inside the Box or Grid and use margins/paddings on these wrappers

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/SignUp/SignUp.tsx]: Separate custom component

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/SignUp/SignUp.tsx]: These TextFields should be inside the Grid. 
```
<Grid container={true} spacing={3}>
  <Grid item={true} xs={12}>
    <TextField />
  </Grid>
</Grid>
```

Comment (AnvarAka1) [reply src/modules/auth/component/SignUp/SignUp.tsx]: No need in margins

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (AnvarAka1) [inline src/modules/auth/component/SignUp/SignUp.tsx]: Where is onChange? value?
Where is setState for eac the textfield?
And where is onSubmit for the form?

**Review by AnvarAka1 — CHANGES_REQUESTED** [review]
Added some comments. Fix all the issues and then after the next review, you can merge. Now, do not merge.

Great work! A lot of things are done. Keep it up!

File: .eslintrc.cjs
Comment (Azimkhan93) [reply .eslintrc.cjs]: done

File: src/modules/auth/component/Login/Login.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/Login/Login.tsx]: done

File: src/modules/auth/component/Login/Login.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/Login/Login.tsx:48]: done

File: src/modules/auth/component/Login/Login.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/Login/Login.tsx]: done

File: src/modules/auth/component/Login/Login.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/Login/Login.tsx]: done

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/SignUp/SignUp.tsx]: done

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/SignUp/SignUp.tsx]: done

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/SignUp/SignUp.tsx]: done

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/SignUp/SignUp.tsx]: done

File: src/modules/auth/component/SignUp/SignUp.tsx
Comment (Azimkhan93) [reply src/modules/auth/component/SignUp/SignUp.tsx]: done

**Review by AnvarAka1 — APPROVED** [review]
Keep it up! You only forgot to add the onSubmit and onChange to the Login.tsx component

**Comment by Azimkhan93** [comment]
> Keep it up! You only forgot to add the onSubmit and onChange to the Login.tsx component

done

**Pull request merged**
